CONTRIBUTING.md

```diff
@@ -75,16 +75,20 @@ In your local repository:
   as previous section requires. You may find this
   [Re-sync Fork With Upstream guide](https://gist.github.com/OleksiyRudenko/8b3ddb664308de0634b53c525e551d8b)
   helpful to fix the issue.
-- ask your peers for review in
-  [FE p2p chat](https://t.me/joinchat/???????????????),
+
+
+### Codereview
+- you will **require** approvals* from **at least two peers**, so seek
+for sufficient support. 
+  - ask your peers for review in [FE Students chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897),
   post a link to your PR as well
+  - if in some reason there are not reviwers from p2p course, you can ask your friends outside of the course<br>
+- **you should provide a review** for this same task to **at least 2 PR's** of other students of p2p course
+- after finishing previous steps ask mentors for review in [FE Questionarium chat](https://web.telegram.org/#/im?p=s1495296464_549300875144563897)
```

Comment:
Is it about a **real** review or we just want someone to merge the PR?

Reply:
@Oodmincheg Please, reflect on the comment above.

Reply:
I thought that minimenotors will do some reviews, and sometimes mentors

Reply:
Clear, thanks. This gonna happen at the beginning for sure. We just do not want this on a regular basis, only from time to time.
What do we definitely need mentors for is merging. So student
- can summon a mentor when help is needed (e.g. the contributing student cannot decide on how to implement changes, or student reviewers see the issue in the code but cannot decide on proper way to fix it)
- has to summon a mentor in order to merge a pr

Not sure if item two should be covered in this section.

Please, suggest wordings for the above.

PS Please, do not use term *minimentor* as it sounds a bit underevaluing ) I'd suggest *junior mentor*, which is also questionable but at least it uses commonly accepted in the industry grade denomination.